### PR TITLE
issue #2 Bootstrap Styles Background Updates

### PR DESCRIPTION
**Pull request by eugene-p-dxc**
- Added minimization for css to webpack.
- Made sample page as a part of ditribution.
- Removed inverse navbar from sample page.
- Updated links in sample doc to point to dist folder(for easier testing)
- navbar/nav changed BG to #38598A. Changed link text decoration
- buttons change the background colour to #38598A for the default and primary, and text colour to white.
- Pills change the background colour to #38598A
- Progress bars Basic and animated background colour changed to  #38598A
- List groups & Primary Panels: background colour changed to  #38598A and text white.

### CODE REVIEW

**Comment by eugene-p-dxc** [comment]
Sorry for multiple commits had to clean up empty lines.
Missed it on the first commit.

**Comment by GregTurner** [comment]
Thanks @eugene-p-dxc! I wasn't expecting this :)

We very likely begin upgrading this repo to v4 in September, hoping the upgrade can take some of the variables with us.  In any case, we can accept this PR for people that want v3 still.

**Comment by eugene-p-dxc** [comment]
I'm aware of v4.  I used my personal account for communication previously @eugene-p. And as we need this repo for our project. Had to create the corporate account.  I'm working on migration of your repo to SASS as well need it sooner. Still, need to figure out few things for a process from our side on how to contribute it back. Do you guys have any preference/requirements/process?

**Comment by eugene-p-dxc** [comment]
Probably make sense to close issues like that before migration.

**Comment by GregTurner** [comment]
Lol, I was just investigating the alternate account when you said that.  

Regarding contributions in v4, it'll be a GitHub/Bootstrap convention.  Use it vanilla and add your custom app stuff to it via SCSS build (trying very hard to follow bootstrap v4 theme conventions).  

If you think the base could be improved, fork it and PR.  

We'll be using the GitHub release feature semver so you can include via package.json like: `mygovbc-bootstrap-theme": "^0.3.0",`  

Unfortunately, I probably have to rename this repo/npm package name to remove the `mygovbc` brand.
